apps/etl/tasks.py

```diff
@@ -21,12 +21,20 @@
 
 
 @shared_task
-def fetch_data():
-    call_command("import_desinventar_data")
-    call_command("import_glide_data")
+def extract_gdacs_data():
     call_command("import_gdacs_data")
 
 
+@shared_task
+def extract_glide_data():
+    call_command("import_glide_data")
```

Comment:
Let's change the commands from `import_` to `extract_` as well.
```suggestion
    call_command("extract_glide_data")
```